No code quickstart (#187)
gatling/frontline-cloud-doc: content/tutorials/no-code/index.md

{{< alert info >}}
**Requirements**
* A Gatling Enterprise Cloud account. Sign up for a [free trial](https://cloud.gatling.io) if you don't already have an account.
{{< /alert >}}

## Inroduction
The Gatling no-code generator is the fastest way to discover load testing and how it can improve your application, microservice, or API. The no-code generator is a graphical user interface that lets you:
- [setup your scenario](#setup-your-scenario),
- [setup the virtual user injection profile](#setup-the-injection-profile),
- (optional) [define acceptance criteria](#define-acceptance-criteria-optional),
- [select a testing location](#select-the-testing-location).

Once you start your simulation, the load testing data are displayed in real-time. The following guide assists you in writing and launching your first load test. To keep learning about Gatling and load testing, see these helpful resources:
- [Gatling Community](https://community.gatling.io)
- [Gatling Academy](https://gatling.io/academy/)

## Access Gatling Enterprise Cloud
To access the Gatling no-code generator:
1. Navigate to https://cloud.gatling.io in your web browser.
2. Login or register if you don't have an account. 

> 3. Click on **Create a new NO CODE simulation** in the _Simulation runs_ pane on the landing page. <!--TO BE Confirmed-->

Author: ```suggestion
3. Click on **Create a simulation without coding** in the _Latest simulation runs_ pane on the landing page. 
```

Author: updated from the testing hour version today

> The no-code generator is divided into 4 steps with a dynamically updated code example in Java, Scala, and Kotlin. The code example lets you see how changes in the simulation settings can be written in your preferred programming language.  

Reviewer: The phrasing seems a bit weird to me ?
Suggestion:

```suggestion
The no-code generator is divided into 4 steps with a dynamically updated code example in Java, Scala, and Kotlin. The code example shows you how the resulting simulation can be written in your preferred programming language.  
```
